bigtable/tests/unit/test_table.py

```diff
@@ -1152,9 +1161,9 @@ def test_callable_no_retry_strategy(self):
         with mock.patch("google.cloud.bigtable.table.wrap_method") as patched:
             patched.return_value = mock.Mock(return_value=[response])
 
-            worker = self._make_worker(client, table.name, [row_1, row_2, row_3])
+            worker = self._make_worker(client, table.name, [[row_1, row_2, row_3]])
```

Comment:
Why do we need second array dimension here?

Reply:
Now, we are passing batches not list of rows